internal/support/scheduler/schedulecontext.go

```diff
@@ -58,7 +62,7 @@ func (sc *IntervalContext) Reset(interval models.Interval, lc logger.LoggingClie
 		// use max time
 		sc.EndTime = time.Unix(1<<63-62135596801, 999999999)
 	} else {
-		t, err := time.Parse(TIMELAYOUT, sc.Interval.End)
+		t, err := time.Parse(SchedulerTimeFormat, sc.Interval.End)
 		if err != nil {
 			lc.Error("parse time error, the original time string is : " + sc.Interval.End)
 		}
```

Comment:
This file `internal/support/scheduler/schedulecontext.go`  only for V1, it can be removed.

Reply:
removed